src/Microsoft.SqlTools.ServiceLayer/TableDesigner/DesignerPathUtils.cs

```diff
@@ -16,7 +16,7 @@ public static class DesignerPathUtils
         /// Below are the 3 scenarios and their expected path.
         /// Note: 'index-{x}' in the description below are numbers represent the index of the object in the list.
 		/// 1. 'Add' scenario
-		///     a. ['propertyName1']. Example: add a column to the columns property: ['columns'].
+		///     a. ['propertyName1',index-1]. Example: add a column to the columns property: ['columns', 0].
 		///     b. ['propertyName1',index-1,'propertyName2']. Example: add a column mapping to the first foreign key: ['foreignKeys',0,'mappings'].
```

Comment:
we also need to add an index at the end, ADS will also provide an index for ADD operation.